layouts/partials/widgets/search.html

```diff
@@ -6,7 +6,7 @@
 <section class='widget widget-search sep-after'>
   <header>
     <h4 class='title widget-title'>
-      {{- $config.title | default ( i18n "search" ) -}}
+      {{- $config.Title | default ( i18n "search" ) -}}
```

Comment:
this should be reverted

Reply:
Reverted in https://github.com/MunifTanjim/minimo/pull/297/commits/aeb5bca7b5d1ec7e7f41de497ad233f13660c45c